opendevin/server/session.py

```diff
@@ -66,13 +62,16 @@ async def start_listening(self):
                     elif action == "chat":
                         self.controller.add_history(NullAction(), UserMessageObservation(data["message"]))
                     else:
+# <<<<<<< patch-1
+                        await self.send_error("Unsupported action")
+  =======
                         await self.send_error("I didn't recognize this action:" + action)
+# >>>>>>> main
```

Comment:
@ImSuvodeep The conflict wasn't properly resolved

Reply:
> @ImSuvodeep The conflict wasn't properly resolved

let me know if the new commit solves ur issue